source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -572,10 +595,12 @@ FilterDataStatus Filter::onData(ProcessorState& state, Buffer::Instance& data, b
   }
 
   if (state.callbackState() == ProcessorState::CallbackState::HeadersCallback) {
-    if (state.bodyMode() == ProcessingMode::STREAMED &&
-        config_->sendBodyWithoutWaitingForHeaderResponse()) {
+    if ((state.bodyMode() == ProcessingMode::STREAMED &&
+         config_->sendBodyWithoutWaitingForHeaderResponse()) ||
+        state.bodyMode() == ProcessingMode::BIDIRECTIONAL_STREAMED) {
       ENVOY_LOG(trace, "Sending body data even header processing is still in progress as body mode "
-                       "is STREAMED and send_body_without_waiting_for_header_response is enabled");
+                       "is BIDIRECTIONAL_STREAMED or STREAMED and "
+                       "send_body_without_waiting_for_header_response is enabled");
```

Comment:
Agree that we should adding some methods to handle individual cases.  However, I would prefer not to do this refactoring within this PR.

WDYT? @tyxia

Reply:
STREAMED and FULL_DUPLEX_STREAMED has a lot of similarity. Adding the if check to skip some code is to avoid a lot of duplicated code. We can tackle the refactoring in a separate PR.

Reply:
Please add a todo in the code, for tracking this then

Reply:
Added a TODO

Reply:
Thanks, please also associated the TODO with your github name? 

I know at google it will pop up the error if it is not associated with bug/ldap.